### PR TITLE
todoist: add page

### PR DESCRIPTION
**Pull request by iuliandita**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/todoist.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+- Add a task:
+
+`todoist add "{{task_name}}"`
+
+- Add a high priority task with a label, project, and due date:
+
+`todoist add "{{task_name}}" --priority {{1}} --label-ids "{{label_id}}" --project-name "{{project_name}}" --date "{{tmr 9am}}"`
+
+- Add the same task, in quick mode:
```
Comment (marchersimon) [inline pages/common/todoist.md:14]: We try to keep the examples as independent as possible from each other.
```suggestion
- Add a high priority task with a label, project, and due date in quick mode:
```

**Review by marchersimon — COMMENTED** [review]
Looks great. Are there maybe commands to authenticate with your Todoist account? That would be useful. Also a command to start it in interactive mode would be great.

**Comment by iuliandita** [comment]
> Looks great. Are there maybe commands to authenticate with your Todoist account? That would be useful. Also a command to start it in interactive mode would be great.

On first run, the app asks for the Todoist token. I am not aware of any command to start it in interactive mode.

File: pages/common/todoist.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+> Access Todoist from the command-line.
+> More information: <https://github.com/sachaos/todoist>.
+
+- Add a task:
+
+`todoist add "{{task_name}}"`
+
+- Add a high priority task with a label, project, and due date:
+
+`todoist add "{{task_name}}" --priority {{1}} --label-ids "{{label_id}}" --project-name "{{project_name}}" --date "{{tmr 9am}}"`
```
[the remark this answers is not in the document]
Comment (iuliandita) [reply pages/common/todoist.md:12]: These flags are working with the latest version, just tested before submitting this PR.
Are you checking this Github page?
https://github.com/sachaos/todoist

File: pages/common/todoist.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+`todoist quick '#{{project_name}} "{{tmr 9am}}" p{{1}} {{task_name}} @{{label_name}}'`
+
+- List all tasks with a header and color:
+
+`todoist --header --color list`
+
+- List all high priority tasks:
+
+`todoist --header --color list --filter p{{1}}`
```
Comment (blueskyson) [inline pages/common/todoist.md:24]: Does every `todoist list` requires option `--header --color`? I think showing `--header --color` in one example is enough.

Comment (iuliandita) [reply pages/common/todoist.md:24]: no, there's no need for that, you can even set up color to be always on in the config...

**Review by blueskyson — APPROVED** [review]

File: pages/common/todoist.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (marchersimon) [reply pages/common/todoist.md:12]: I was on this page. None of the option are listed there. The repo probably needs a PR with the up-to-date `--help` output.

**Review by marchersimon — APPROVED** [review]
Thanks @iuliandita

File: pages/common/todoist.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (iuliandita) [reply pages/common/todoist.md:12]: Ah, I see. Indeed, those flags don't show up using --help. I'll see if I can make a PR for todoist to update --help. Thanks!

**Pull request merged**